straxen/plugins/event_patternfit.py

```diff
@@ -19,7 +19,7 @@
     strax.Option('s2_tf_model', help='S2 (x, y) optical data-driven model', infer_type=False,
                  default='XENONnT_s2_optical_map_data_driven_ML_v0_2021_11_25.tar.gz'),
     strax.Option('s1_aft_map', help='Date drive S1 area fraction top map.', infer_type=False,
-                 default='s1_aft_dd_xyz_XENONnT_Kr83m_41500eV_31Oct2021.json'),
+                 default='s1_aft_dd_xyz_XENONnT_kr-83m_08Feb2022.json'),
```

Comment:
```suggestion
```

Reply:
Thanks Yossi, but this suggestions confuses me now :D Since we want to use CMT I think your upper example is correct and therefore we drop this option completely right?

Reply:
yeah, the entire "Option()" needs to be removed and replaced by a URLConfig class attribute.  I had to split my suggested change into two suggestions due to githubs limitations on suggesting changes on existing changes.
The map loading also needs to be removed from the `setup()` method.